pkg/cmd/generate/templates.go

```diff
@@ -15,57 +15,39 @@ var (
 		SERVICE_REGISTRY_CORE_PATH=` + registrycmdutil.REGISTRY_CORE_PATH + `
 		SERVICE_REGISTRY_COMPAT_PATH=` + registrycmdutil.REGISTRY_COMPAT_PATH + `
 		{{end}}
-		## Authentication Configuration
-		RHOAS_CLIENT_ID={{.ClientID}}
-		RHOAS_CLIENT_SECRET={{.ClientSecret}}
-		RHOAS_OAUTH_TOKEN_URL={{.TokenURL}}
 `)
 
 	templateJSON = heredoc.Doc(`
 		{
 			{{if .KafkaHost}}"kafkaHost":"{{.KafkaHost}}",
 			{{end}}{{if .RegistryURL}}"serviceRegistryUrl":"{{.RegistryURL}}",
 			"serviceRegistryCorePath":"` + registrycmdutil.REGISTRY_CORE_PATH + `",
-			"serviceRegistryCompatPath":"` + registrycmdutil.REGISTRY_COMPAT_PATH + `",
-			{{end}}"rhoasClientID":"{{.ClientID}}",
-			"rhoasClientSecret":"{{.ClientSecret}}",
-			"rhoasOauthTokenUrl":"{{.TokenURL}}"
+			"serviceRegistryCompatPath":"` + registrycmdutil.REGISTRY_COMPAT_PATH + `"{{end}}
 		}
 	`)
 
 	templateProperties = heredoc.Doc(`
 		## Generated by rhoas cli
 		{{if .KafkaHost}}## Kafka Configuration
 		kafkaHost={{.KafkaHost}} 
-		{{end}}{{if .RegistryURL}}	## Service Registry Configuration
+		{{end}}{{if .RegistryURL}}## Service Registry Configuration
 		serviceRegistryUrl={{.RegistryURL}}
 		serviceRegistryCorePath=` + registrycmdutil.REGISTRY_CORE_PATH + `
-		serviceRegistryCompatPath=` + registrycmdutil.REGISTRY_COMPAT_PATH + `
-		{{end}}
-		## Authentication Configuration
-		rhoasClientID={{.ClientID}}
-		rhoasClientSecret={{.ClientSecret}}
-		rhoasOauthTokenUrl={{.TokenURL}}
+		serviceRegistryCompatPath=` + registrycmdutil.REGISTRY_COMPAT_PATH + `{{end}}
 	`)
 
-	templateSecret = heredoc.Doc(`
+	templateConfigMap = heredoc.Doc(`
```

Comment:
@wtrocki 
This PR changes `generate-config` command to skip creating a service account. And replaces secret with configmap.